polkadot-parachains/src/chain_spec.rs

```diff
@@ -202,6 +202,9 @@ fn integritee_genesis_config(
 		aura: parachain_runtime::AuraConfig { authorities: initial_authorities },
 		aura_ext: Default::default(),
 		parachain_system: Default::default(),
+		teerex: TeerexConfig { allow_sgx_debug_mode: true },
+		claims: Default::default(),
+		treasury: Default::default(),
```

Comment:
Is `Default::default()` the same as the solochain?

Reply:
Yes, it seems so: https://github.com/integritee-network/integritee-node/blob/master/node/src/chain_spec.rs#L392